# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1217903567 14400
#      Mon Aug 04 22:32:47 2008 -0400
# Branch release-3-0-x
# Node ID a0d567862eb80bafbfd2fb153c67571dfeca985f
# Parent  66c8e8252d85b42d185f38294dcdd14b565bd334
oct-env.cc (octave_env::instance_ok): fix type in error message

diff --git a/liboctave/ChangeLog b/liboctave/ChangeLog
--- a/liboctave/ChangeLog
+++ b/liboctave/ChangeLog
@@ -1,8 +1,12 @@
+2008-08-04  John W. Eaton  <jwe@octave.org>
+
+	* oct-env.cc (octave_env::instance_ok): Fix typo in error message.
+
 2008-06-20  Jaroslav Hajek <highegg@gmail.com>
 
 	* MatrixType.h: Add missing include statement.
 
 2008-05-19  David Bateman  <dbateman@free.fr>
 
 	* dSparse.cc: Replace some DGBCON with GPBCON where they are
 	incorrectly used.
diff --git a/liboctave/oct-env.cc b/liboctave/oct-env.cc
--- a/liboctave/oct-env.cc
+++ b/liboctave/oct-env.cc
@@ -84,17 +84,17 @@ octave_env::instance_ok (void)
   bool retval = true;
 
   if (! instance)
     instance = new octave_env ();
 
   if (! instance)
     {
       (*current_liboctave_error_handler)
-	("unable to create current working directoy object!");
+	("unable to create current working directory object!");
 
       retval = false;
     }
 
   return retval;
 }
 
 std::string
